h2o-py/h2o/estimators/gbm.py

```diff
@@ -953,3 +953,10 @@ def check_constant_response(self, check_constant_response):
         self._parms["check_constant_response"] = check_constant_response
 
 
+    """
+      Returns actual number of trees in GBM model (whether CV is enabled or not).
```

Comment:
How is this related to CV?

I think we can mention early stopping instead.

Reply:
It is taken from the ticket description. Will fix.

Reply:
fixed